crates/router/src/connector/aci.rs

```diff
@@ -234,10 +234,29 @@ impl
 
     fn get_url(
         &self,
-        _req: &types::PaymentsAuthorizeRouterData,
+        req: &types::PaymentsAuthorizeRouterData,
         connectors: &settings::Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
-        Ok(format!("{}{}", self.base_url(connectors), "v1/payments"))
+        match req.request.mandate_id.as_ref() {
+            Some(mandate_ids) => {
+                let connector_mandate_id = match mandate_ids.mandate_reference_id.clone() {
+                    Some(api_models::payments::MandateReferenceId::ConnectorMandateId(
+                        connector_mandate_ids,
+                    )) => connector_mandate_ids.connector_mandate_id,
+                    _ => None,
+                };
+                let man_id =
+                    connector_mandate_id.ok_or(errors::ConnectorError::MissingRequiredField {
```

Comment:
add new error called `MissingConnectorMandateId` in errors::ConnectorError and use that here. 
This MissingRequiredField is meant only for customer not proving required fields

Reply:
Done👍